source/administration-guide/manage/admin/content-flagging.rst

```diff
@@ -0,0 +1,90 @@
+Enable Content Flagging
+========================
+
+.. include:: ../../../_static/badges/entry-adv.rst
+  :start-after: :nosearch:
+
+Content Flagging is an :ref:`experimental <administration-guide/manage/feature-labels:experimental>` feature that helps prevent accidental data spillage and helps system administrators respond quickly to potential leaks without disrupting collaboration. Enabling this featureempowers Mattermost users to report messages that may contain sensitive, regulated, or inappropriate information, and enables designated content reviewers to assess and take appropriate action by removing or dismissing flagged messages.
```

Comment:
I don't think this is an experimental feature, actually I'm not sure what qualifies, and when, as an experimental feature. @marianunez d we want to label this as experimental? I'll have to add the experimental tag in System Console UI if we want to.

Reply:
Changed Experimental link to MVP

Reply:
This is not an experimental feature, this has been properly validated and tested with security reviews and all the process for GA - see labels definitions here: https://docs.mattermost.com/administration-guide/manage/feature-labels.html

This is indeed an MVP but this is not a label for our feature. It just means the minimum feature set we can deliver that is of value to use.